scripts/zones/AlTaieu/mobs/Jailer_of_Love.lua

```diff
@@ -20,76 +22,283 @@ local minionGroup =
     [7] = 25, -- Qnhpemde
 }
 
-entity.onMobEngage = function(mob, target)
+local spellLists =
+{
+    [1] = 5025, -- Fire List
+    [2] = 5026, -- Ice List
+    [3] = 5027, -- Wind List
+    [4] = 5028, -- Earth List
+    [5] = 5029, -- Ltng List
+    [6] = 5030, -- Water List
+    [7] = 5024, -- Light List
+    [8] = 5031, -- Dark List
+}
+
+-- Animations, action IDs and elemental absorb mods are directly mapped to eachother per retail caps
+-- in the "standard" order, Fire Ice Wind Earth Thunder Water Holy Dark
+local eleAbsorbModID      = { 459, 460, 461, 462, 463, 464, 465, 466 }
```

Comment:
This concrete line is unnecesary, I think. We have the element table inside combat folder for them, following the same elemental order

Reply:
removed and replaced wit the combat definition